code/game/area/areas/shuttles.dm

```diff
@@ -106,19 +106,19 @@
 
 /area/shuttle/pod_1
 	name = "Escape Pod One"
-	area_flags = BLOBS_ALLOWED
+	area_flags = BLOBS_ALLOWED | CULT_PERMITTED
```

Comment:
pretty sure this was one of the areas explicitly we dont want cult in

Reply:
then why can blob start their core there? I was following that train of thought

Reply:
If I recall correctly, cult were originally stopped from making runes on shuttles, to stop them from making a bunch of runes on the whiteship, and then warping to station to do shit.

Having them be able to do so in pods, given they only work once, seems harmless enough

Reply:
Escape Pods allowing runes is probably a small issue compared to mining survival pods, since you can set up mining pods literally anywhere.

Reply:
its only survival pods, not whiteship so I dunno, pods are also tiny as fuck, max 3 tiles

Reply:
> its only survival pods, not whiteship so I dunno, pods are also tiny as fuck, max 3 tiles

can't you decon the walls of the pod to get more space?

and since tele runes are a thing, it doesn't really matter WHERE in lavaland the pod crashes, as long as it's not too close to the mining base or something.

I'd recommend not letting cults OR blobs set up in an escape pod, for the same reason why we don't let them set up in a shuttle or deep in lavaland.

Reply:
i agree remove this